rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderTests.java

```diff
@@ -243,7 +242,7 @@ private void testGetOldFederation(
     }
 
     @Test
-    void saveNewFederation_before_RSKIP123_should_allow_to_save_any_fed_type() throws IOException {
+    void saveNewFederation_before_RSKIP123_should_allow_to_save_any_fed_type() {
         testSaveNewFederation(
```

Comment:
Why not just create a parametrized test?

Reply:
That's another story, the intention here is to simply fix the failing tests